reserved_keyword_checker/tests/test_app/test_app/settings.py

```diff
@@ -0,0 +1,114 @@
+"""
+Django settings for test_app project.
+
+Generated by 'django-admin startproject' using Django 1.11.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'qmq2luug748jlqfv9a5ep18h39z6zkapr4%%r7=zdn9f=o00in'
```

Comment:
Nit: Make this look more fake so that people's security alarms don't go off when they read it?

Reply:
Good call.